modules/virt-installing-a-watchdog.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog.adoc
+
+[id="virt-defining-a-watchdog_{context}"]
+= Installing a watchdog device
+
+To activate a watchdog device, install the `watchdog` package on your virtual machine and start the watchdog service.
+
+.Procedure
+
+. Install the watchdog package and dependencies:
```

Comment:
I would mention that you have to be root because most of our commands allow you to be another user.

`As root user, install the `watchdog` package:`

Reply:
@apinnick Should watchdog be marked that way in all instances?

Reply:
https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

See second row, second bullet of table.